src/utils/sign-images-in-ans-object/index.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import signImagesInANSObject from ".";
+
+<Meta title="Utilities/Sign Images In ANS Object" component={signImagesInANSObject} />
+
+# Sign Images In ANS Object
+
+The `signImagesInANSObject()` function is a support function for `content-source` requests. It returns a "thenable" function that accepts an `axios` [response object](https://axios-http.com/docs/res_schema), and inspects the `data` value from the returned object. It then finds any image objects within the `data` structure that do not have an `auth` token, issues a [cachedCall request](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/developer-documentation/partial-caching-with-pagebuilder-engine/) to get an `auth` token for the resizer, and returns the ANS Object transformed to include this newly generated key.
+
+## Usage
+
+```jsx
+import axios from "axios";
+
+import { ARC_ACCESS_TOKEN, RESIZER_APP_VERSION } from "fusion:environment";
+import { fetch as resizerFetch } from "@wpmedia/signing-service-content-source";
+
+import signImagesInANSObject from "./signImagesInANSObject";
+
+const fetch = ({ ... }, { cachedCall }) => {
+	return axios({
+		method: "GET",
+		url,
+		headers: {
+    		"content-type": "application/json",
+    		Authorization: `Bearer ${ARC_ACCESS_TOKEN}`
+    	},
+	})
+	.then(signImagesInANSObject(cachedCall, resizerFetch, RESIZER_APP_VERSION))
```

Comment:
This is the only "edge case" territory that worries me. We may want to expand this `.then` logic in data sources as `signImagesInANSObject` becomes a critical dependency. If there's a failure in a signing an image, the `promise.all` results in no data being returned by the content source. Seems like we need a fallback where the feed data is preserved, but the image data is either omitted or replaced with the fallback image.

Still, I don't think this would be in the scope of this ticket.